code/modules/antagonists/gang/gang_themes.dm

```diff
@@ -0,0 +1,276 @@
+/datum/gang_theme
+	var/name = "Gang Theme"
+	var/list/involved_gangs = list()
+	var/description = "I dunno, some shit here."
+	var/list/gang_objectives = list() // assoc list, type = "objective"
+	var/list/bonus_items = list() // Items given to every gangster in this theme.
+	var/list/bonus_first_gangster_items = list() // Stuff given to the starting gangster at roundstart. Assoc list, type = list(item_type)
+	var/list/everyone_objective = null // If this isn't null, everyone gets this objective.
+
+/datum/gang_theme/proc/get_gangs()
+	return involved_gangs
```

Comment:
Avoid getter procs, as per the contribution guidelines.
Store this kind of information in a variable.